packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -114,9 +114,24 @@ class PictureLayer extends Layer {
   /// The picture's coodinate system matches this layer's coodinate system
   ui.Picture picture;
 
+  /// Hints that the painting in this layer is complex and would benefit from
+  /// caching.
+  ///
+  /// If this hint is not set, the compositor will apply its own huerstics to
+  /// decide whether the this layer is complex enough to benefit from caching.
+  bool isComplex = false;
```

Comment:
`isComplex` is sort of nebulous. How about `cacheContentHint` or something generally more descriptive. Just a thought.